python/src/scipp/plotting/resampling_model.py

```diff
@@ -126,16 +130,13 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         # TODO See #1469. This is a temporary hack to work around the
         # conversion of coords to edges in model.py.
-        new_meta = {name: var for name, var in self._array.meta.items()}
-        self._array = PlotArray(data=self._array.data,
-                                masks=self._array.masks,
-                                meta=new_meta)
-        for name, var in self._array.meta.items():
+        self._array = self._array.copy()
```

Comment:
Why deep copy?

Reply:
I wanted to shallow copy, and i was thinking of python dicts, and forgot that we decided to deep copy by default.